package.json

```diff
@@ -57,6 +57,7 @@
     "@types/lodash.isequal": "^4.5.5",
     "@types/node": "^12.7.4",
     "all-contributors-cli": "^6.9.0",
+    "dotenv": "^8.1.0",
```

Comment:
remove `dotenv`. You can use other tools. For example, https://github.com/direnv/direnv

Reply:
I use dotenv for my own projects when I'm using this library. But I don't see the need for it here since there is not enough env variables for a project level dotenv or similar.